apps/rebar/test/rebar_dir_SUITE.erl

```diff
@@ -68,8 +68,8 @@ end_per_testcase(_, _Config) -> ok.
 default_src_dirs(Config) ->
     {ok, State} = rebar_test_utils:run_and_check(Config, [], ["compile"], return),
 
-    [] = rebar_dir:src_dirs(rebar_state:opts(State)),
-    ["src"] = rebar_dir:src_dirs(rebar_state:opts(State), ["src"]).
+    ["src"] = rebar_dir:src_dirs(rebar_state:opts(State)),
```

Comment:
```suggestion
    [] = rebar_dir:src_dirs(rebar_state:opts(State), []),
    ["src"] = rebar_dir:src_dirs(rebar_state:opts(State)),
```
That way we make sure it's still possible to get functional behavior for empty values.